keyboards/tada68/tada68.h

```diff
@@ -37,6 +37,8 @@
     {k40, k41, k42, XXX, XXX, k45, XXX, XXX, XXX, k49, k4a, k4b, k4c, k4d, k4e}  \
 }
 
+#define LAYOUT_65_ansi LAYOUT_ansi
```

Comment:
Swap the names around:
```suggestion
#define LAYOUT_ansi LAYOUT_65_ansi
```
and do the needful above.